packages/pangea-sdk/pangea/tools_util.py

```diff
@@ -104,12 +117,16 @@ def json_defaults(obj):
         return str(obj)
 
 
-def filter_deep_none(data: dict) -> dict:
-    return {
-        k: v if not isinstance(v, dict) else filter_deep_none(v)
-        for k, v in data.items()
-        if v is not None
-    }
+def filter_deep_none(data: t.Dict) -> t.Dict:
+    return {k: v if not isinstance(v, t.Dict) else filter_deep_none(v) for k, v in data.items() if v is not None}
+
+
+def get_test_domain(environment: TestEnvironment):
+    return os.getenv(f"PANGEA_INTEGRATION_DOMAIN_{environment}")
```

Comment:
nit: I prefer just failing loudly in cases where environment variables don't exist and they're needed. makes debugging easier

Reply:
Agree. I'll add that